src/site/content/en/lighthouse-seo/robots-txt/index.md

```diff
@@ -2,34 +2,36 @@
 layout: post
 title: robots.txt is not valid
 description: |
-  Learn about robots-txt audit.
+  Learn about the "robots.txt is not valid" Lighthouse audit.
+updated: 2019-08-21
 web_lighthouse:
   - robots-txt
 ---
 
-The `robots.txt` file tells search engines
-what pages of your site they can crawl.
-An invalid `robots.txt` configuration can cause
-2 general types of problems:
+The `robots.txt` file tells search engines which of your site's pages they can
+crawl. An invalid `robots.txt` configuration can cause two types of problems:
 
-- Not crawling public pages, causing your relevant content
-to show up less in search results.
-- Crawling private pages,
-exposing private information in search results.
+- It can keep search engines from crawling public pages, causing your
+  content to show up less often in search results.
+- It can cause search engines to crawl pages you may not want shown in search
+  results.
+
+## How the Lighthouse `robots.txt` audit fails
```

Comment:
`robots.txt` is formatted as inline code. I don't think you've been formatting any of the other H2s as inline code.

Reply:
I say omit it, because it sometimes looks weird to have inline code formatting in the section title

Reply:
Actually, I've been using code font in headings when needed, as have various contributors. Are you saying we should disallow code font in headings? That rule seems like it could get in the way of good semantics to meet an aesthetic preference?